sdk/keyvault/keyvault-keys/samples/typescript/src/cryptography.ts

```diff
@@ -29,6 +29,7 @@ export async function main(): Promise<void> {
   // Connection to Azure Key Vault Cryptography functionality
   const myWorkKey = await client.createKey(keyName, "RSA");
 
+  // Creating a CryptographyClient from the identifier (or URL) of a KeyVaultKey
```

Comment:
The only difference between this and the new sample is that we are passing different types to the constructor. Is there really a net benefit in creating a whole new sample for it? Is it not enough to document this option?

For example, here we can say

> You can use either the key or the key Id to create a CryptographyClient

And then we can update the documentation on for the first parameter to the constructor to denote that both values are supported.

Reply:
This works for me! Thank you.

Reply:
I believe I did what you requested. Please let me know if you wanted something different.